playbook/spec/pb_kits/playbook/kits/legend_spec.rb

```diff
@@ -23,8 +23,9 @@
 
   describe "#classname" do
     it "returns namespaced class name", :aggregate_failures do
-      color = "data_1"
-      expect(subject.new(color: color, dark: true, text: "Text").classname).to eq "pb_legend_kit_data_1 dark"
+      expect(subject.new(text: "Test default").classname).to eq "pb_legend_kit_data_1"
+      expect(subject.new(color: "data_5", dark: true, text: "Text").classname).to eq "pb_legend_kit_data_5 dark"
+      expect(subject.new(color: "success", text: "Test").classname).to eq "pb_legend_kit_success"
```

Comment:
this expectation could probably use some love / clarification. We're not necessarily checking that the correct classname is applied in every single instance, but instead in named instances -- so it may be prudent to clarify that in the specs and check them singly.